### PR TITLE
fixes #13157 - adding padding to icons

### PR DESCRIPTION
**Pull request by unorthodoxgeek**

### CODE REVIEW

**Comment by tbrisker** [comment]
This does not fix the original bug but rather adds even more padding.

**Comment by unorthodoxgeek** [comment]
@tbrisker - did I not understand the issue correctly? to my understanding the issue requests more padding, so it looks like it did before

**Comment by domcleal** [comment]
Sorry if it wasn't clear, but @tbrisker's reading of the ticket is correct - the padding should be reduced to what it was, except on the host#show page between the icon and text.  fa2a0c76 affected more than the icon/text padding on that one page, and should not have affected padding to the left of the icon or on host#index etc.

**Comment by unorthodoxgeek** [comment]
ok, I see what I got wrong, updated.

**Comment by unorthodoxgeek** [comment]
@tbrisker can you take a look please?

File: app/assets/stylesheets/patternfly_and_overrides.scss [hunk 1]
```diff
@@ -291,7 +291,9 @@ a.btn-info span{
   padding-bottom: 4px;
 }
 
-span.status-error, span.status-ok, span.status-warn, span.status-question {
-  padding-left: 3px;
+#properties_table {
```
Comment (tbrisker) [inline app/assets/stylesheets/patternfly_and_overrides.scss]: Please move this to hosts.scss, this has nothing to do with patternfly overrides.

File: app/assets/stylesheets/patternfly_and_overrides.scss [hunk 2]
```diff
@@ -291,7 +291,9 @@ a.btn-info span{
...
+  span.status-error, span.status-ok, span.status-warn, span.status-question {
```
Comment (tbrisker) [inline app/assets/stylesheets/patternfly_and_overrides.scss]: This also adds the padding to the text, which has the same class. Why not just `.host-status` which will select just the icon?

**Comment by tbrisker** [comment]
@unorthodoxgeek There are two more rules for `.host-status` in `application.scss` that are not really used anymore as they don't match the current pages, please remove them.

**Comment by tbrisker** [comment]
Merged as a80e058901dd9bfc0e75b13963b2dc33123918f3, thanks @unorthodoxgeek!

**Pull request closed without merging**